src/extensions/ratchet_tree_extension.rs

```diff
@@ -35,7 +35,6 @@ pub struct RatchetTreeExtension {
     tree: Vec<Option<Node>>,
 }
 
-// TODO #186: Implement group extensions
```

Comment:
Shall we leave this in then until the issue and https://github.com/mlswg/mls-protocol/pull/464 are resolved?

Reply:
Yes, #186 is essentially blocked by that.